tasks/connectors/bugcrowd/bugcrowd_task.rb

```diff
@@ -181,21 +181,22 @@ def extract_finding(issue)
       def extract_definition(issue)
         {
           "name" => issue["attributes"]["vrt_id"],
-          "description" => CGI.escapeHTML("#{Sanitize.fragment(issue['attributes']['title'])}\n\n#{Sanitize.fragment(issue['attributes']['description'])}"),
           "solution" => issue["attributes"]["remediation_advice"],
```

Comment:
@jscalvo I think the solution need to move too unless you can see in the data that it is common across items. from the data we looked at it was unique to the finding. This would leave the vuln_def with only the name and scanner_type